astropy/stats/sigma_clipping.py

```diff
@@ -6,6 +6,14 @@
 import warnings
 from ..utils.exceptions import AstropyDeprecationWarning, AstropyUserWarning
 
+try:
+    from scipy.stats import sigmaclip
+    HAS_SCIPY=True
+except:
+    warnings.warn("Scipy is required for some functionality of",
```

Comment:
Since the message continues in the next line, there should be no coma on the end of this line